src/header/banner.scss

```diff
@@ -0,0 +1,23 @@
+.banner-wrapper {
+  background: rgba(200,200,200,0.3);
+  border-bottom: 1px solid #C8C8C8;
+  font-family: 'Benton', sans-serif !important;
```

Comment:
normally you want to avoid `!important` by making this more specific. because of the multiple uses of this stylesheet across platforms, and since the markup will stay the same, i'd just double up on this by putting it into the `.banner h1` below.